src/lib.rs

```diff
@@ -32,8 +32,8 @@ pub fn is_uri_trustworthy(uri: &str) -> bool {
         && uri.bytes().next().unwrap().is_ascii_alphabetic()
         && uri
             .bytes()
-            .take(10)
-            .take_while(|b| b.is_ascii_alphanumeric() || [b'+', b'-', b':'].contains(b))
+            .take(15)
+            .take_while(|b| b.is_ascii_alphanumeric() || [b'+', b'-', b':', b'.'].contains(b))
```

Comment:
Can you update the doc-comment above with the new requirements.

Reply:
Sorry, somehow I missed it.

The PR is updated.